src/Assets/TestProjects/WatchAppWithProjectDeps/AppWithDeps/Program.cs

```diff
@@ -16,7 +16,12 @@ public static void Main(string[] args)
             Console.WriteLine("Started");
             // Process ID is insufficient because PID's may be reused.
             Console.WriteLine($"Process identifier = {Process.GetCurrentProcess().Id}, {Process.GetCurrentProcess().StartTime:hh:mm:ss.FF}");
-            Thread.Sleep(Timeout.Infinite);
+
+            while (true)
+            {
+                Lib.Print();
```

Comment:
out of curiosity, what does this Print statement do?  (maybe doc?)

Reply:
This is a test project. Lib is F# class with method Print on it.

Reply:
oh, sorry :)